supertokens_python/supertokens.py

```diff
@@ -147,33 +157,43 @@ def defaultImpl(o: Any):
         return json.dumps(self, default=defaultImpl, sort_keys=True, indent=4)
 
 
-def manage_cookies_post_response(session: SessionContainer, response: BaseResponse):
+def manage_session_post_response(session: SessionContainer, response: BaseResponse):
```

Comment:
the impl of this should be to just loop through the session.method_to_call and pass in the response to it